trikot-viewmodels-declarative/compose/src/main/kotlin/com/mirego/trikot/viewmodels/declarative/compose/viewmodel/VMDImage.kt

```diff
@@ -37,15 +37,14 @@ fun VMDImage(
     placeholderContentScale: ContentScale = contentScale,
     alpha: Float = DefaultAlpha,
     colorFilter: ColorFilter? = null,
-    contentDescription: String? = null,
     allowHardware: Boolean = true,
     placeholder: @Composable ((placeholderImageResource: VMDImageResource, state: AsyncImagePainter.State) -> Unit) = { imageResource, state ->
         RemoteImageDefaultPlaceholder(
             imageResource = imageResource,
             modifier = modifier,
             contentScale = placeholderContentScale,
             colorFilter = colorFilter,
-            contentDescription = contentDescription
+            contentDescription = viewModel.contentDescription
```

Comment:
If we are to use one of the viewModel field here in the placeholder, I guess it would by wise to observeAsState it.